pants.toml

```diff
@@ -77,6 +77,7 @@ interpreter_constraints = [">=3.7,<3.10"]
 
 [docformatter]
 args = ["--wrap-summaries=100", "--wrap-descriptions=100"]
+experimental_lockfile = "src/python/pants/backend/python/lint/docformatter/lockfile.txt"
```

Comment:
Bit of a tangent (because this needs to be nested more deeply because we re-generate it when we change the default), but: there are lots of potential conventions to be formed, but something like `3rdparty/lock/${resolve_name}.*.txt` is likely to make the most sense for end users.

For performance reasons, end users are really going to want to share resolves within the repo rather than leaning into a pattern where every single binary has their own unique resolve. Since we're not anticipating "hundreds" of them (a handful, most likely... at most a dozen?) centralizing them seems like it encourages that.